src/main/resources/com/google/api/codegen/js/sample.snip

```diff
@@ -10,6 +10,10 @@
          https://console.developers.google.com/apis/api/{@class.apiName}
       2. Get access keys for your application. See
          https://developers.google.com/api-client-library/javascript/start/start-js#get-access-keys-for-your-application
+      @if class.auth.instructionsUrl
+        3. For additional information on making an authenticated call, see
```

Comment:
One reason I wanted to be more specific wrt to saying "call" was that the instructions might include more information than just the auth stuff. For example, it's a bit of a pain to run a JS sample because it has to come from `localhost`, so a local server has to be run, and the OAuth setup has to point to that local server...

See the JS instructions for `sheets:v4`: https://developers.google.com/sheets/api/quickstart/js#step_2_set_up_the_sample

I'm not sure if replacing with "authentication" would convey that the site includes more information.